docs/security/permissions.md

```diff
@@ -126,6 +126,11 @@ You can't remove or delete the default server level groups.
 				and not user accounts or groupsthat contain user accounts.
 				By default, this group is a member of <strong>Team Foundation Administrators</strong>.
 			</p>
+			<p>
+				If you need to add an account to this group after you install TFS, you can do so using
+				the TFSSecurity.exe utility in the <strong>Tools</strong> subfolder of your TFS installation directory.
```

Comment:
I don't like the `<strong>` tags, but it seems that the \` for inline code doesn't work in this context (or at least github isn't rendering it)... If the \` works for inline code rendering, please change on merge.

Reply:
thanks! ` does indeed work when rendered without our content pipeline. i'll update the merge, and thank you for your contribution!